integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDynamicUpdate.java

```diff
@@ -246,6 +247,14 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
         + "            ListenPort : 8001";
 
     assertDoesNotThrow(() -> Files.write(pathToAddClusterYaml, yamlToAddCluster.getBytes()));
+
+    // write sparse yaml to change ScatteredReadsEnabled for adminserver
+    pathToChangReadsYaml = Paths.get(WORK_DIR + "/changereads.yaml");
+    String yamlToChangeReads = "topology:\n"
+        + "    Server:\n"
+        + "        \"admin-server\":\n"
+        + "            ScatteredReadsEnabled: true";
```

Comment:
Why do we need this admin server attribute.  Is it MUST for dynamic change feature ?

Reply:
this is one of the non-dynamic change that's done in the tests